core/code/chat.js

```diff
@@ -1,3 +1,5 @@
+/* global TEAM_TO_CSS */
```

Comment:
These are defined under window. We should imho rather use e.g. `window.teamToStringId` than keep defining these in files again